### PR TITLE
Add African pidgin language translation

### PR DESCRIPTION
**Pull request by LizzyOrji123**
I created a new language called pidgin and created best-practice.md and other files.

The goal of my branch is to encourage Africans to join the open source community by writing in a language every African can understand which is Pidgin. I would love to recreate the entire guidelines in Pidgin. Thank you.… added all best practices in pidgin language

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by LizzyOrji123** [comment]
If you speak pidgin and would love to contribute to the pidgin open-source on Github. I would appreciate your help.

**Comment by ahpook** [comment]
@LizzyOrji123 Hi, are you still interested in making this contribution? I left a review with a few requested changes - I'd love to get this merged! Please take a look.

File: _data/locales/pidgin.yml
[the remark this answers is not in the document]
Comment (LizzyOrji123) [reply _data/locales/pidgin.yml:1]: Sorry for all the delay, i am finishing up this project. thank you, i would change to pcm.yml

**Comment by LizzyOrji123** [comment]
> @LizzyOrji123 Hi, are you still interested in making this contribution? I left a review with a few requested changes - I'd love to get this merged! Please take a look.

I have made the corrections. Thank you for being so patient and detailed with your work.

File: assets/css/translate.scss [hunk 1]
```diff
@@ -27,6 +27,7 @@ $section_translation: (
         "tr": "",
         "zh-hant": "章節",
         "zh-hans": "章节",
+        "pidgin": "Portion",
```
Comment (ahpook) [inline assets/css/translate.scss:30]:
```suggestion
        "pcm": "Portion",
```

File: _articles/pcm/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
+lang: pidgin
```
Comment (ahpook) [inline _articles/pcm/code-of-conduct.md:2]:
```suggestion
---
lang: pcm
```

File: _articles/pcm/finding-users.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
+---
+lang: pidgin
```
Comment (ahpook) [inline _articles/pcm/finding-users.md:2]:
```suggestion
---
lang: pcm
```

**Review by ahpook — APPROVED** [review]
Looks great @LizzyOrji123 ! Thanks so much. I made a couple of small suggestions in code review so that the metadata will work with the site generator - I think once those changes are in we'll be good to merge!

**Comment by ahpook** [comment]
Very happy to get this merged in. This is now LIVE on the site! You can see it here

https://opensource.guide/pcm/

**Pull request merged**